aixcalibuha/sensitivity_analysis/sobol.py

```diff
@@ -68,13 +85,301 @@ def generate_samples(self):
                             N=self.num_samples,
                             **self.create_sampler_demand())
 
-    def _get_res_dict(self, result: dict, cal_class: CalibrationClass):
+    def _save(self, result, time_dependent=False):
+        if time_dependent:
```

Comment:
Why no `super()._save`?

Reply:
I haven't thought about it here. I have added it here.